apps/files/tests/test_tasks.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf8 -*-
+import mock
+from datetime import datetime, timedelta
+
+from django.conf import settings
+
+import amo.tests
+from files import tasks
+from files.models import File
+from versions.models import Version
+
+
+# Very basic js file that contains a `let`, a `const`, and a `var` in the
+# toplevel scope. Both the `let` and `const` should be rewritten to `var`.
+TEST_JS_FILE = 'apps/files/fixtures/files/test_with_toplevel_let.js'
+
+
+def assert_test_file_fixed(filename):
+    """Check that the content of the "test file" has been fixed."""
+    with open(filename, 'r') as fixed_js_file:
+        # The "let foo = 1;" has been "fixed" to "var foo = 1".
+        assert fixed_js_file.read() == (
+            "var foo = 1;\nvar   bar = 2;\nvar baz = 3;\n")
```

Comment:
I'm a fan of

```python
expected = "var foo = 1;\nvar   bar = 2;\nvar baz = 3;\n"
assert fixed_js_file.read() == expected
```

instead of breaking lines here.